docs/campaign.rst

```diff
@@ -93,15 +93,21 @@ For instance, given the set of nodes defined above, we can define the
           - srv2
       gpu:
         match: gpu-.*
+      all-cpus:
+        constraint: skylake
 
-Both methods are being used:
+All methods are being used:
 
 * **nodes** expects an exhaustive list of nodes. The
   `ClusterShell <http://clustershell.readthedocs.io/en/latest/tools/nodeset.html#usage-basics>`_
   `NodeSet` syntax is also supported.
 
 * **match** expects a valid regular expression
 
+* **constraint** expects a string. This tag does not references node
+  names explicitely but instead delegates it to SLURM. Value of constraint tag
+  is given to the sbatch options thru the *--constraint* option.
```

Comment:
(nitpicking again...)
thru --> through
and just the line above:
_The_ value of _the_ constraint tag